NOTICE.txt

```diff
@@ -506,7 +601,7 @@ THE SOFTWARE.
 
 --------------------------------------------------------------------
 Dependency: github.com/bsm/sarama-cluster
-Revision: 7e67d87a6b3f83fe08c096fd084691bd9dca112f
+Revision: incompatible
```

Comment:
Any idea why we get 'incompatible' here? Can we fix it easily by upgrading this dependency (or removing it in case it is not needed anymore)?

Reply:
Oh, this is a mistake on my side. Let me fix it.